src/styled/styled.test.tsx

```diff
@@ -33,7 +33,7 @@ describe('styled() API', () => {
     expect(getByTestId('button').dataset.type).toBe('secondary');
   });
 
-  it('should pass styles from styled', () => {
+  it('should pass styles from tasty', () => {
```

Comment:
I guess this was replaced automatically and there should be `styled`?

Reply:
yep